tests/common.py

```diff
@@ -57,6 +58,14 @@
 ENGLISH_ASR_MODEL_ID = "asr-wav2vec2-base-960h-english"
 
 
+def get_status_message(status: Status):
+    message = f"{status.code} {status.description}"
+    if status.details:
+        return f"{message} {status.details}"
+    else:
+        return message
```

Comment:
Minor: Else isn't needed here

Reply:
Logically this is the code that has two paths.  I intentionally use if/else to match what the code is doing and how it looks like.  


Pattern like
```python
if ...: 
  return ...
other_code
```

Is used for early escape patterns.  This kind o similar to the 2 paths, but I prefer it when one path is short, and other is much bigger.